app/settings.py

```diff
@@ -12,11 +12,19 @@ def get_key(key_name):
 RABBIT_URL = 'amqp://{user}:{password}@{hostname}:{port}/{vhost}'.format(
     hostname=os.getenv('SEFT_RABBITMQ_HOST', 'localhost'),
     port=os.getenv('SEFT_RABBITMQ_PORT', 5672),
-    user=os.getenv('SEFT_RABBITMQ_DEFAULT_USER', 'rabbit'),
-    password=os.getenv('SEFT_RABBITMQ_DEFAULT_PASS', 'rabbit'),
+    user=os.getenv('SEFT_RABBITMQ_DEFAULT_USER', 'admin'),
```

Comment:
Shouldn't have to prefix every env var with `SEFT` - though may be required until config is separate per-service